src/Jackett.Common/Models/IndexerConfig/Bespoke/ConfigurationDataImmortalSeed.cs

```diff
@@ -0,0 +1,15 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace Jackett.Common.Models.IndexerConfig.Bespoke
+{
+    [ExcludeFromCodeCoverage]
+    internal class ConfigurationDataImmortalSeed : ConfigurationDataBasicLogin
+    {
+        public BoolConfigurationItem FreeleechOnly { get; private set; }
+
+        public ConfigurationDataImmortalSeed()
+        {
+            FreeleechOnly = new BoolConfigurationItem("Search freeleech only") { Value = false };
```

Comment:
Worth adding that this overrides the `Sort by` setting?
It's annoying that this renders the indexer basically useless for *arrs' RSS sync.

Same situation for XSpeeds?